app/actions/remote/channel.ts

```diff
@@ -339,7 +339,7 @@ export const joinChannel = async (serverUrl: string, userId: string, teamId: str
             channel = await client.getChannel(channelId);
         } else if (channelName) {
             channel = await client.getChannelByName(teamId, channelName, true);
-            if ([General.GM_CHANNEL, General.DM_CHANNEL].includes(channel.type)) {
+            if ([General.GM_CHANNEL, General.DM_CHANNEL].includes(channel.type as any)) {
```

Comment:
```suggestion
            if ([General.GM_CHANNEL, General.DM_CHANNEL].includes(channel.type as string)) {
```

I know the actual type is `ChannelType` but in this case I rather cast it as a string instead of any

Reply:
The problem here is that the type of the array is `Array<"G"|"D">` instead of `Array<string>`. I made a small adjustment to not use the `as any`.